docs/src/opf/acp.md

```diff
@@ -1,3 +1,149 @@
 # AC-OPF
 
-See [`ACPPowerModel`](https://lanl-ansi.github.io/PowerModels.jl/stable/formulation-details/#PowerModels.ACPPowerModel) formulation in [`PowerModels.jl`](https://lanl-ansi.github.io/PowerModels.jl/stable/).
\ No newline at end of file
+
+## Mathematical Formulation
+
+The ACOPF model considered in OPFGenerator is presented below.
+
+```math
+\begin{align}
+    \min_{\PG, \QG, \PF, \QF, \PT, \QT, \VM, \VA} \quad
+    & \sum_{i \in \NODES} \sum_{j \in \GENERATORS_{i}} c_j \PG_j \label{model:acopf:obj} \\
+    \text{s.t.} \quad \quad \quad
+    & \sum_{j\in\GENERATORS_i}\PG_j - \sum_{j\in\LOADS_i}\PD_j - \GS_i \VM_i^2 = \sum_{e \in \mathcal{E}_{i}}  \PF_{e} + \sum_{e \in \mathcal{E}^R_{i}} \PT_{e}
+        & \forall i &\in \NODES
+        \label{model:acopf:kirchhoff:active} \\
+    & \sum_{j\in\GENERATORS_i}\QG_j - \sum_{j\in\LOADS_i}\QD_j + \BS_i \VM_i^2 = \sum_{e \in \mathcal{E}_{i}} \QF_{e} + \sum_{e \in \mathcal{E}^R_{i}} \QT_{e}
+        & \forall i &\in \NODES
+        \label{model:acopf:kirchhoff:reactive} \\
+    & \PF_{e} = \gff_{e}\VM_i^2 + \gft_{e} \VM_i \VM_j \cos(\VA_i-\VA_j) + \bft_{e} \VM_i \VM_j \sin(\VA_i-\VA_j)
+        & \forall e = (i,j) &\in \EDGES
+        \label{model:acopf:ohm:active:from} \\
+    & \QF_{e} = -\bff_{e} \VM_i^2 - \bft_{e}\VM_i \VM_j \cos(\VA_i-\VA_j) + \gft_{e} \VM_i \VM_j \sin(\VA_i-\VA_j)
+        & \forall e = (i,j) &\in \EDGES
+        \label{model:acopf:ohm:reactive:from} \\
+    & \PT_{e} = \gtt_{e}\VM_j^2 + \gtf_{e} \VM_i \VM_j \cos(\VA_i-\VA_j) - \btf_{e} \VM_i \VM_j \sin(\VA_i-\VA_j)
+        & \forall e = (i,j) &\in \EDGES
+        \label{model:acopf:ohm:active:to} \\
+    & \QT_{e} = -\btt_{e} \VM_j^2 - \btf_{e}\VM_i \VM_j \cos(\VA_i-\VA_j) - \gtf_{e} \VM_i \VM_j \sin(\VA_i-\VA_j)
+        & \forall e = (i,j) &\in \EDGES
+        \label{model:acopf:ohm:reactive:to} \\
+    & (\PF_{e})^2 + (\QF_{e})^2 \leq \overline{S_{e}}^2
+        & \forall e &\in \EDGES
+        \label{model:acopf:thrmbound:from} \\
+    & (\PT_{e})^2 + (\QT_{e})^2 \leq \overline{S_{e}}^2
+        & \forall e &\in \EDGES
+        \label{model:acopf:thrmbound:to} \\
+    & \dvamin_{e} \leq \VA_i - \VA_j \leq \dvamax_{e}
+        & \forall e = (i,j) &\in \EDGES
+        \label{model:acopf:angledifference} \\
+    & \VA_\text{ref} = 0
+        \label{model:acopf:slackbus} \\
+    & \pgmin_{i} \leq \PG_i \leq \pgmax_{i}
+        & \forall i &\in \GENERATORS
+        \label{model:acopf:pgbound} \\
+    & \qgmin_{i} \leq \QG_i \leq \qgmax_{i}
+        & \forall i &\in \GENERATORS
+        \label{model:acopf:qgbound} \\
+    & \underline{\VM_i} \leq \VM_i \leq \overline{\VM_i}
+        & \forall i &\in \NODES
+        \label{model:acopf:vmbound} \\
+    & {-}\overline{S_{e}} \leq  \PF_{e} \leq \overline{S_{e}}
```

Comment:
under/overline only on the symbol, excluding the sub/super script to match the other conventions
```suggestion
    & {-}\overline{S}_{e} \leq  \PF_{e} \leq \overline{S}_{e}
```